portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,40 @@
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<meta charset="utf-8">
+<title>My Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<div id="header-content">
+  <div id="image-container">
+    <img src="images/main-selfie.jpg" alt="Picture of me" id="header-selfie">
+    <button id="cycle-button" onclick="cyclePictures()"> Cycle Pictures </button>
+  </div>	
+  <div id="greeting-container">
+    <h1>Welcome to my portfolio</h1>
+    <h3>My name is Malachi Brewer. I am a sophomore at the University of Georgia.</h3>
+    <h3>Welcome Buds! I created this site to house random pieces of information about me for anyone who might be curious. 
+            Clicking on the buttons below will take you to different parts of the page.</h3> 
+    <ul id="header-nav"> 
+      <li><a href="#about-me">About Me</a>
+      <li><a href="#education">Education</a>
+      <li><a href="#professional">Professional</a>
+      <li><a href="#random-facts">Random Facts</a>
+    </ul>
+  </div>
+  </div>
+    <h1 id="about-me"> About Me </h1> 
+    <p> I was gifted to this world on June 7, 2001 in Clevalnd, Ohio. My family moved away from Ohio when I was around 16 months old.
+        Since we arrived in Georgia, we have also moved around several times. Currently we are living in a neighboorhead around 45 miles north of Atlanta. </p>
+    <h1 id="education"> Education </h1> 
+    <p> Although I didn't always like school growing up, I have also enjoyed learning new things about the world and other people
+        which pushed to start appreciating my education more once I got to high school. The subjects that I liked the most was always math or science.
+        I graduated high school in summer 2019 and am heading into my second year as a copmuter science major at the University of Georgia. </p>
+    <h1 id="professional"> Professional </h1>
+    <p><a href="https://www.linkedin.com/in/malachi-brewer-48957b15b"> LinkedIn </a></p>
+    <p><a href="https://github.com/malachibre"> GitHub </a></p>
+    <p> I first started working when I was 17 years old at Banana Republic. I stayed for a year until I left for college. 
+	At UGA, I started a job working for the web team for the College of Agricultre and Enviornmental Science. 
+        I had to pause my work here to have time for my STEP internship over the summer, and will resume working there in August. </p>
+    <h1 id="random-facts"> Random Facts </h1>
+    <p> Whenever someone asks for a fun fact, my go to response is how I've broken both of my arms. Once when I was on a skateboard and once when I was on a bike.
+        One thing that others point in me is how I can remember the tiniest things.
```

Comment:
Remove the single whitespace characters immediately following and preceding the `>` and `<` characters, respectively.

Reply:
Done.